src/libraries/Common/src/System/IO/Archiving.Utils.Windows.cs

```diff
@@ -69,11 +69,8 @@ public static unsafe string EntryFromPath(ReadOnlySpan<char> path, bool appendPa
                     string.Empty;
             }
 
-#pragma warning disable CS8500 // takes address of managed type
-            ReadOnlySpan<char> tmpPath = path; // avoid address exposing the span and impacting the other code in the method that uses it
-            return string.Create(appendPathSeparator ? tmpPath.Length + 1 : tmpPath.Length, (appendPathSeparator, RosPtr: (IntPtr)(&tmpPath)), static (dest, state) =>
+            return string.Create(appendPathSeparator ? tmpPath.Length + 1 : tmpPath.Length, (appendPathSeparator, path), static (dest, state) =>
```

Comment:
This case is unfortunate since tuple hasn't employed `allow ref struct`.

Reply:
It can't. It's not a ref struct.

Reply:
Yeah, it lacks a feature to say "be ref struct if any of type argument is ref struct". We don't create a `RefStructTuple` either.